onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3792,12 +3807,14 @@ def aten_native_channel_shuffle(self: TensorType, groups: int) -> TensorType:
     raise NotImplementedError()
 
 
+@torch_op("aten::native_dropout")
 def aten_native_dropout(
-    input: TensorType, p: float, train: Optional[bool]
-) -> tuple[TensorType, TensorType]:
+    input: TTensor, p: float, train: bool = True
+) -> tuple[TTensor, TTensor]:
```

Comment:
```suggestion
) -> Tuple[TTensor, TTensor]:
```

this will fix the py38 error